vue/src/store.ts

```diff
@@ -635,13 +635,11 @@ async function queryModelRuns(type: 'firstPage' | 'nextPage', filters: QueryArgu
  */
 const refreshModelRunDownloadingStatuses = async () => {
   const queryState = state.queryStates.loadModelRuns;
-  const loadedPages = queryState.nextPage - 1;
   const activeFilters = queryState.activeFilters;
-
   let existingRequest = queryState.request as CancelablePromise<void> | undefined;
 
   // chain cancelable requests
-  for (let page = 0; page < loadedPages; page++) {
+  for (let page = 1; page < queryState.nextPage; page++) {
```

Comment:
Yep, I have this fixed in #509, but I like using `queryState.nextPage` instead (I was initially concerned with race conditions updating it while this task ran, but I'm less worried about that now.) I'll remove it from my PR and have it updated here.